### DISCUSSION TITLE
hide certain cams

### DISCUSSION THREAD

**Discussion by alkralor** [blakeblackshear/frigate#6951] [Ask A Question]
Hi, 
I have frigate installed on HassOS.. I have 3 cameras and since my HA is accessed by my household, there is one cam that I would prefer was not accessed by everyone.

However, frigate does not seem to let me delegate cams to specific users, and it puts all the recorded files together in the media folder.

Is what I want even possible? To seclude one of the cameras to record to a separate, encrypted folder or something.

Thanks.

**Comment by NickM-27** [comment]
This is not currently possible
